modules/drivers/mongo/src/metabase/driver/mongo/query_processor.clj

```diff
@@ -507,16 +508,18 @@
     (throw (ex-info (tru "now is not supported for MongoDB versions before 4.2")
                     {:database-version (get-mongo-version)}))))
 
-(defmethod ->rvalue :datetime-add        [[_ inp amount unit]] (do
-                                                                 (check-date-operations-supported)
-                                                                 {"$dateAdd" {:startDate (->rvalue inp)
-                                                                              :unit      unit
-                                                                              :amount    amount}}))
-(defmethod ->rvalue :datetime-subtract   [[_ inp amount unit]] (do
-                                                                 (check-date-operations-supported)
-                                                                 {"$dateSubtract" {:startDate (->rvalue inp)
-                                                                                   :unit      unit
-                                                                                   :amount    amount}}))
+(defmethod ->rvalue :datetime-add [[_ inp amount unit]]
+  (check-date-operations-supported)
+  {"$dateAdd" {:startDate (->rvalue inp)
+               :unit      unit
+               :amount    amount}})
+
+(defmethod ->rvalue :datetime-subtract
+  [[_ inp amount unit]]
+  (check-date-operations-supported)
+  {"$dateSubtract" {:startDate (->rvalue inp)
+                    :unit      unit
+                    :amount    amount}})
```

Comment:
Minor formatting tweaks. Not sure why Kondo didn't catch the redundant `do` here in the past but I fixed it either way